Assertion Error on Windows.

### Assertion Error on Windows

Getting AssertionError on Windows due to the backslashes in the Windows environment. Need to normalize the URL.

Work-around: Don't use windows :) (Still looking for another work-around)

``` 
    AssertionError: Provided start directory is not within root or one of its subdirectories.
```


I added the following logs to the file:
``` 
/node_modules/bundalo/node_modules/file-resolver/lib/util.js
```
```js
exports.locate = function locate(name, root, start) {
    var removed, file, parent;

console.log("------------");
console.log("start:");
console.log(start);
console.log("------------");
console.log("root:");
console.log(root);
console.log("------------");
    assert(~start.indexOf(root), 'Provided start directory is not within root or one of its subdirectories.');
```
Output of the above logs:
```
------------
start:
locales\US\en
------------
root:
locales/
------------
```

Replicated with the Test Suite:
```
# resolve with a bogus locale object

assert.js:86
  throw new assert.AssertionError({
        ^
AssertionError: Provided start directory is not within root or one of its subdir
ectories.
    at Object.locate (Z:\Projects\file-resolver\lib\util.js:64:5)
    at Object.proto._locate (Z:\Projects\file-resolver\index.js:33:24)
    at Object.proto.resolve (Z:\Projects\file-resolver\index.js:51:26)
    at Test.<anonymous> (Z:\Projects\file-resolver\test\index.js:116:28)
    at Test.bound [as _cb] (Z:\Projects\file-resolver\node_modules\tape\lib\test
.js:62:32)
    at Test.run (Z:\Projects\file-resolver\node_modules\tape\lib\test.js:75:10)
    at Test.bound [as run] (Z:\Projects\file-resolver\node_modules\tape\lib\test
.js:62:32)
    at Test._end (Z:\Projects\file-resolver\node_modules\tape\lib\test.js:141:11
)
    at Test.bound [as _end] (Z:\Projects\file-resolver\node_modules\tape\lib\tes
t.js:62:32)
    at Test.<anonymous> (Z:\Projects\file-resolver\node_modules\tape\lib\test.js
:140:40)
npm ERR! Test failed.  See above for more details.```

--- a/lib/util.js
+++ b/lib/util.js
@@ -59,6 +59,8 @@ exports.parseLangTag = function (str) {
 exports.locate = function locate(name, root, start) {
     var removed, file, parent;
 
+    start = path.normalize(start);
+    root = path.normalize(start);
     assert(~start.indexOf(root), 'Provided start directory is not within root or one of its subdirectories.');
 
     file = path.join(start, name);

